src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -575,6 +575,29 @@
         return getCallbackRespEntityErrors(errors, caseData);
     }
 
+    @PostMapping(value = "/addAmendJurisdiction", consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "amend jurisdiction details for a single case.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Accessed successfully", response = CCDCallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request"),
+        @ApiResponse(code = 500, message = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> addAmendJurisdiction(
+            @RequestBody CCDRequest ccdRequest,
+            @RequestHeader(value = "Authorization") String userToken) {
+        log.info("AMEND JURISDICTION ---> " + LOG_MESSAGE + ccdRequest.getCaseDetails().getCaseId());
```

Comment:
Please correct me if I am wrong but I think we are auto populating conciliation track here and not amending jurisdiction? Can an existing callback be used for it rather than creating a new call back?